### ISSUE TITLE
V9: Fix A & AA accessibility issues throughout Backoffice

### ISSUE THREAD

**Issue by nielslyngsoe** [umbraco/Umbraco-CMS#12546]
There are currently a large number of accessibility issues throughout the Backoffice.
We have decided to collect the issues on a separate tracker to provide an overview of the current state.

Issues:
https://github.com/umbraco/Umbraco-CMS.Accessibility.Issues/issues

Status board:
https://github.com/orgs/umbraco/projects/30/views/1

**How to grab a task**
* Have a look through the table above and pick an issue you'd like to fix.
* Write a comment that you are looking into it
* Fix the issue 😄
* Submit a PR on the [CMS repository](https://github.com/umbraco/Umbraco-CMS) with a reference to the issue from the [Accessibility repository](https://github.com/umbraco/Umbraco-CMS.Accessibility.Issues/issues) by writing `Fixes umbraco/Umbraco-CMS.Accessibility.Issues#1234` somewhere in the PR comment.

**Fixing an issued**
Here are some links to various tools and reference materiel should you require any help 🙂

Websites & Articles
[Web Content Accessibility Guidelines (WCAG) Overview](https://www.w3.org/WAI/standards-guidelines/wcag/)
[Getting Comfortable with WCAG](https://seesparkbox.com/foundry/getting_comfortable_with_wcag)
[Accessibility according to actual people with disabilities](https://axesslab.com/accessibility-according-to-pwd/)
[Colorblind Accessibility on the Web – Fail and Success Cases](https://axesslab.com/colorblind-accessibility-web-fail-success-cases/)
[AngularJS guide to accessibility](https://docs.angularjs.org/guide/accessibility)
Tools

[Accessibility Insights - Chrome Extension](https://chrome.google.com/webstore/detail/accessibility-insights-fo/pbjjkligggfmakdaogkfomddhfmpjeni)
[Axe: Accessibility for web development](https://www.deque.com/axe/)
[Color Contrast Checker](https://webaim.org/resources/contrastchecker)
Activate built in screen reader

Mac - toggle Voice Over by pressing Command-F5
PC- toggle Narrator by pressing Ctrl + Winkey +Enter

If you have any questions you can reach out either here or on the Umbraco Community #accessibility Slack channel. You can join the community slack channel by going to [umbracians.chat](http://umbracians.chat/).

Big thanks to the wonderful Danny Lancaster who has created the accessibility audit report which the Accessibility issues originate from. 

Ps. If you find something that is not mentioned please report it, thanks

**Comment by nul800sebastiaan** [comment]
Pro tip, in your commit message, on a new line, writing the same notation works too:

`Fixes umbraco/Umbraco-CMS.Accessibility.Issues#1234`
